tools/tf2ngraph.py

```diff
@@ -343,7 +421,16 @@ def main():
     args = prepare_argparser(allowed_formats)
     inp_format, inp_loc = filter_dict("input", args.__dict__)
     out_format, out_loc = filter_dict("output", args.__dict__)
-    output_nodes = args.output_nodes.split(',')
+    if args.output_nodes is None:
+        possible_out_nodes = infer_output_nodes(inp_format, inp_loc)
+        print(
+            "Analysed graph for possible list of output nodes. Please supply one or more output node in --output_nodes"
+        )
+        for out_node in possible_out_nodes:
+            print(out_node, "of type", possible_out_nodes[out_node])
+        raise ValueError("No output node name provided in --output_nodes")
```

Comment:
Don't throw exception as it leads to printing the stack trace - which makes it hard to understand the error message.  Follow the error reporting style from this script: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/tools/import_pb_to_tensorboard.py

Reply:
Sorry, I don't see the error reporting you mention.

the most related thing in that script is:
```
try:
  from tensorflow.contrib.tensorrt.ops.gen_trt_engine_op import *
except ImportError:
  pass
```


I can rewrite the function to not throw an error, but just print something, but then if tf2ngraph is in an automated script, it is much easier to detect the thrown error, than to parse the stdout for prints saying that it failed.

Reply:
Replaced `assert`s and `raise`s with `exit_on_error`